Description: Spelling errors in binary and in HISTORY file.
 Correct spelling error of the word 'overridden' at line 1480 of the HISTORY file.
 Correct spelling error of the word 'overridden' at line 198 of mmail/resource.cc file.
 Correct spelling error of the word 'Suppress' at line 243 of mmail/resource.cc file.
Bug: https://github.com/wmcbrine/MultiMail/issues/2
Author: Robert James Clay <jame@rocasa.us>
Origin: vendor
Last-Update: <2015-09-22>

--- a/HISTORY
+++ b/HISTORY
@@ -1477,7 +1477,7 @@
   area or message. The default character set is CP437 (as before), except
   for Internet or Usenet areas (as marked by Blue Wave or QWKE flags), which
   are Latin-1. The character set in individual Fidonet messages can be
-  overriden by the "CHRS:" or "CHARSET:" kludges. Replies are set according
+  overridden by the "CHRS:" or "CHARSET:" kludges. Replies are set according
   to the default for the destination area.
 
 - The console character set and the default sort types for the packet and
--- a/mmail/resource.cc
+++ b/mmail/resource.cc
@@ -195,7 +195,7 @@
  "(shown here) will be used.",
  "",
  "If you change either of the base directories, all the subsequent paths",
- "will be changed, unless they're overriden in the individual settings.",
+ "will be changed, unless they're overridden in the individual settings.",
  "",
  "Please see the man page for a more thorough explanation of these options.",
  0
@@ -240,7 +240,7 @@
  "8-bit character set for SOUP packets (see docs)",
  "Quoted-printable options for outgoing messages (see docs)",
     0, 0, 0,
- "Supress help messages (use more of the screen for content)",
+ "Suppress help messages (use more of the screen for content)",
  "For QWK only: Generate indexes from MESSAGES.DAT instead of *.NDX",
  "Allow use of the mouse?"
 #ifdef USE_SPAWNO
